libraries/SITL/SIM_EFI_Hirth.h

```diff
@@ -0,0 +1,240 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  simulate Hirth EFI system
+
+./Tools/autotest/sim_vehicle.py --gdb --debug -v ArduPlane -A --uartF=sim:hirth --speedup=1
+param set SERIAL5_PROTOCOL 24
+param set SIM_EFI_TYPE 6
+param set EFI_TYPE 6
+reboot
+status EFI_STATUS
+
+./Tools/autotest/autotest.py --gdb --debug build.Plane test.Plane.Hirth
+
```

Comment:
Want to drop a link to the manufacturer's docs so one could cross-reference against the packet definitions?